### DISCUSSION TITLE
Appropriate Figure and Table numbering in appendices

### DISCUSSION THREAD

**Discussion by kellijohnson-NOAA** [pfmc-assessments/sa4ss#111] [Show and tell]
Hi all 👋. First, I want to thank @iantaylor-NOAA for asking me a question about how to add an Appendix to his document. Second, I would like to highlight the below example of how to appropriately add an Appendix to your document. You need to add the following to your document before the header of your first appendix.
```
\counterwithin{table}{section}
\counterwithin{figure}{section}
# (APPENDIX) Appendix {-}
```

🎵 that I have not tested this out within a {sa4ss} document and rather just within a simple reproducible example. So, feel free to leave comments if it is not working for you and we can troubleshoot it. Below is a link to the results from my simple example and the code to create the pdf, though for the code you will need to change the extension to run it because GitHub would not let me upload a .Rmd file.
```
bookdown::render_book("app_test.Rmd")
```
[_main.pdf](https://github.com/pfmc-assessments/sa4ss/files/13529523/_main.pdf)
[app_test.txt](https://github.com/pfmc-assessments/sa4ss/files/13529616/app_test.txt)

**Comment by iantaylor-NOAA** [comment]
This worked perfectly for me, thank you @kellijohnson-NOAA.
Main text fig and table numbering was unchanged but appendix now has Table A.1 and Figure A.1 etc.

**Comment by kellijohnson-NOAA** [comment]
Here is an [issue in the quarto repository](https://github.com/quarto-dev/quarto-cli/discussions/4581) regarding how to add appendices to a quarto 📜 just in case we eventually go down that road.
